src/pages/patterns/filtering/index.mdx

```diff
@@ -69,6 +69,62 @@ Multiple category selection is most commonly manifested in either a vertical lis
 </Column>
 </Row>
 
+### Advanced filtering
+
+_Advanced search_ can be used when the user requires to search and filter a list of items by choosing more than one filter from multiple filter categories. This pattern is especially useful when the number of items within each category is dynamic, such as filtering a table by a user or user defined tags.
+
+Advanced search is also commonly used when there is not the requried amount of space on the screen to accommodate a vertical list.
+
+<br />
+<Row>
+<Column colLg={8}>
+
+![Advanced filter input field example](images/filter-8.png)
+
+</Column>
+</Row>
+<br />
+<br />
+
+The filtering categories appear as headers within a dropdown space underneath a search field, with the top five items from within each section appearing as defaults within this space, along with the expected number of returns if the user was to select that option.
+
+<br />
+<Row>
+<Column colLg={8}>
+
+![Example of dropdown with suggested filters being shown](images/filter-9.png)
+
+</Column>
+</Row>
+<br />
+<br />
+
+As the user types into the search field the categories of filters are themselves filtered by the text entered, with the matching text being highlighted.
+
+<br />
+<Row>
+<Column colLg={8}>
+
+![The dropdown with a text filter applied](images/filter-10.png)
+
+</Column>
+</Row>
+<br />
+<br />
+
+Entering a search term which exactly matches an item within a drop down enters that filter into the search field as a tag with the appropriate label attached. Alternatively a user can select that item directly from the drop down by clicking on it. If the term entered into the search field doesn't match any values within the filter categories then that term is entered as a free text search.
```

Comment:
replace with:
Entering a search term which exactly matches an item within a drop down enters that filter into the search field as a tag with the appropriate label attached; a user may also select that item directly from the drop down by clicking on it. If the term entered into the search field doesn't match any values within the filter categories, that term is entered as a free text search, not in a tag.